src/couch_mrview_util.erl

```diff
@@ -212,8 +212,11 @@ init_state(Db, Fd, State, #index_header{
     purge_seq=PurgeSeq,
     id_btree_state=IdBtreeState,
     view_states=ViewStates}) ->
-    NewViewStates = lists:map(fun({Pos, {Cnts, UsrReds, _DtSize}, Size}) ->
-        {{Pos, {Cnts, UsrReds}, Size}, nil, nil, Seq, PurgeSeq}
+    NewViewStates = lists:map(fun
+        (nil) ->
```

Comment:
I don't think we would ever have nil here.
1. We would hit [this first](https://github.com/cloudant/couchdb-couch-mrview/blob/57882-view-conversion-bug/src/couch_mrview_util.erl#L208)
2. Then we get [here](https://github.com/cloudant/couchdb-couch-mrview/blob/57882-view-conversion-bug/src/couch_mrview_util.erl#L197)
3. Finally we initialize [view_states](https://github.com/cloudant/couchdb-couch-mrview/blob/57882-view-conversion-bug/src/couch_mrview_util.erl#L203)